main.go

```diff
@@ -132,6 +132,6 @@ func main() {
 
 	log.Print("Initializing API...")
 	api.Initialize()
-	log.Print("Running web server...")
+	log.Printf("Running web server at: http://127.0.0.1%s ...", server.Addr)
```

Comment:
Should we not just have the IP specified (as 0.0.0.0) in the server initialization? Then it'll go into the string anyways?